src/uct/api/uct.h

```diff
@@ -2022,6 +2094,22 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
 ucs_status_t uct_ep_create(const uct_ep_params_t *params, uct_ep_h *ep_p);
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Connect an endpoint after it is bound to a network device.
```

Comment:
used only by the client side, right? need to state which side should call this function pls.

Reply:
actually it depends on implementation.. we can implement this for server as well as a replacement of pack callback but it looks redundant at least for our known use cases.